requirements-py3.txt

```diff
@@ -2,16 +2,16 @@ Cython>=0.23.5
 decorator>=4.0.9
 future>=0.16.0
 h5py>=2.5.0
-libtiff>=0.4.2
+#libtiff>=0.4.2
 matplotlib>=1.4.3,<3.0
 numpy>=1.11.0,<1.17
-opencv>=2.4.11,<4.0
-Pillow>=3.2.0
-pycairo>=1.10.0
+#opencv>=2.4.11,<4.0
+#Pillow>=3.2.0
+#pycairo>=1.10.0
 PyInstaller>=3.6
 psutil>=4.3.0
 PyYAML>=3.11
 pyzmq>=15.2.0
 scipy>=0.15.1
-wxPython>=4.0.4
+#wxPython>=4.0.4

```

Comment:
All these dependencies are needed by Odemis, so there is no good reason to comment them out.